canvases/dark-packages/main.dark

```diff
@@ -0,0 +1,35 @@
+type PackageFns = PACKAGE.Darklang.Stdlib.ProgramTypes.PackageFn.T
+
+[<DB>]
+type PackageFnsDB = PackageFns
```

Comment:
Using
```
[<DB>]
type PackageFnsDB = PACKAGE.Darklang.Stdlib.ProgramTypes.PackageFn.T
```
instead of 
```
type PackageFns = PACKAGE.Darklang.Stdlib.ProgramTypes.PackageFn.T

[<DB>]
type PackageFnsDB = PackageFns
```
didn't work

Reply:
Could you add a commented failing test case for this?

Reply:
possibly as a followup.